fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -318,8 +318,10 @@
      * The connection timeout and socket timeout config for thrift server
      * The value for thrift_client_timeout_ms is set to be larger than zero to prevent
      * some hang up problems in java.net.SocketInputStream.socketRead0
+     * 
+     * make it one hour to avoid fe close the socket so frequently, that be will retry to cost more time
      */
-    @ConfField public static int thrift_client_timeout_ms = 30000;
```

Comment:
when will fe close the socket frequently?

Reply:
in our case, query information_schema.tables.  i find fe would send a TCP RST command(try tcpdump 9020 in fe) to be to close the socket becasuse of thrift_client_timeout_ms is meet, the be log comes "no more data to read", the be would retry in 1s(default).